apps/mark-scan/backend/src/custom-paper-handler/state_machine.test.ts

```diff
@@ -500,22 +497,28 @@ test('voting flow happy path', async () => {
   await waitForStatus('not_accepting_paper');
 });
 
-describe('removing ballot during presentation state', () => {
-  test('goes to ballot_removed_during_presentation state', async () => {
-    await executePrintBallotAndAssert(
-      ballotPdfData,
-      scannedBallotFixtureFilepaths
-    );
+test('ballot invalidation flow', async () => {
```

Comment:
Moved this here from `app.test`, to make it easier to reuse the utilities here (and since it's mostly testing state machine logic)